PrivacyPolicy.md

```diff
@@ -0,0 +1,50 @@
+# Information Collected And Transmitted By Git Extensions
+
+First, a reminder: Git Extensions is provided "as is", without warranty of any kind, express or implied, including but not limited to the warranties of merchantability, fitness for a particular purpose and noninfringement.
+
+With that out of the way, here's a breakdown of all the information we may collect via Application Insights.
+
+### Application-Level
+Includes:
+* Exception information
+  * Could, in rare cases, contain paths packages on your computer
+* Machine name
+* Host name
+* Version number (e.g. 2.0.x.x)
+* Build type (whether the application is an official release build or not)
+* Selected layout settings (such as visibility of the left panel, commit info position etc)
+* Change of selected layout settings
+
+### Operating System-Level
+Includes:
+* Architecture (e.g. 32-bit)
+* Version (e.g. Windows 10.0.17763.0)
+* Build (e.g. 17134.1.amd64fre.rs4_release.180410-1804)
+* Available processors/cores (e.g. 8 cores)
+* Machine Name (e.g. MyFastPC)
```

Comment:
ditto

Reply:
It isn't something we collect explicitly, I'd think these are collected by one of packages referenced in ApplicationInsights.config.
I'm guessing, it would be `DeviceTelemetryInitializer`

 @onovotny are you able to shed some light?
Is there any way to exclude these extra props we don't require?

Reply:
Those are set in the AppInsights TelemetryClient:
https://github.com/Microsoft/ApplicationInsights-dotnet/blob/80025b5d79cc52485510d422cfa5a0a8159dac83/src/Microsoft.ApplicationInsights/TelemetryClient.cs#L544

Looks like you could manually set RoleInstance to something else using a TelemetryInitalizer and it'd avoid sending it.